src/modules/perceptrons/rblattVectors/RblattVectorsDemo.tsx

```diff
@@ -6,7 +6,13 @@ const RblattVectorsDemo = () => {
     const [brdId, setBrdId] = useState('board_' + Math.random().toString(36).substr(2, 9));
     const [phase, setPhase] = useState(0);
     const [board, setBoard] = useState<any>(null);
-    const [perpPoint, setPerpPoint] = useState<any>(null); 
+    const [perpPoint, setPerpPoint] = useState<any>(null);
+    const phaseMessages = [
+        'The initial classification line is represented by its perpendicular vector Wt.',
+        'After finding a misclassified point, we calculate the distance d between it and the vector\'s origin/',
+        'We then calculate the new vector by subtracting d from Wt.',
```

Comment:
"We then update our vector Wt by subtracting the distance d from the current end point of Wt."